kivy/uix/tabbedpanel.py

```diff
@@ -368,6 +368,24 @@ def get_current_tab(self):
     defaults to 100.
     '''
 
+    bar_width = NumericProperty('2dp')
+    '''Width of the horizontal scroll bar. The width is interpreted
+    as a height for the horizontal bar.
+
```

Comment:
Would you please add `.. versionadded` tags here.

Reply:
Thanks for reviewing.

Is the version 2.2.0 OK?
```
.. versionadded:: 2.2.0
```